### PR TITLE
Add cleanup playbook

### PR DESCRIPTION
**Pull request by vkmc**
Remove created resources for the cluster
- Instances
- Security group
- Networks

TODO. Remove hosts from the Ansible inventory

### CODE REVIEW

File: cleanup.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+# This playbook performs service cleanup
+#
+
+- hosts: localhost
+  connection: local
+  vars_files:
+    - keys.yml
+  tasks:
+    - include: roles/local/tasks/cleanup.yml
```
Comment (tombarron) [inline cleanup.yml]: I changed the name of this role so it should now be:

  - include: roles/openstack_infra/tasks/cleanup.yml

File: roles/openstack_infra/tasks/cleanup.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    wait: yes
+- name: delete custom security group
+  os_security_group:
+    state: absent
+    name: manila-kube-security-group
+- name: delete keypair
+  os_keypair:
+    state: absent
+    name: "{{ key_name }}"
+    public_key_file: "{{ public_key_file }}"
```
Comment (tombarron) [inline roles/openstack_infra/tasks/cleanup.yml]: I don't think we need the public_key_file line for the removal.

File: cleanup.yml (hunk 1 above)
Comment (vkmc) [reply cleanup.yml]: Oops

File: roles/openstack_infra/tasks/cleanup.yml (hunk 2 above)
Comment (vkmc) [reply roles/openstack_infra/tasks/cleanup.yml]: Done

File: roles/openstack_infra/tasks/cleanup.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+- name: delete worker2
+  os_server:
+    name: worker2
+    state: absent
+    wait: yes
+- name: delete custom security group
+  os_security_group:
+    state: absent
+    name: manila-kube-security-group
+- name: delete keypair
```
Comment (gouthampacha) [inline roles/openstack_infra/tasks/cleanup.yml]: don't delete the keypair - it's not created by us here

Comment (vkmc) [reply roles/openstack_infra/tasks/cleanup.yml]: It is created by us https://github.com/tombarron/manila-kube/blob/master/roles/openstack_infra/tasks/main.yml#L34

Comment (gouthampacha) [reply roles/openstack_infra/tasks/cleanup.yml]: I see, but I was using an existing keypair as my "key_name" in keys.yml... If the public key is the same, an existing keypair can be used as is... so let's not delete it, please :)

Comment (vkmc) [reply roles/openstack_infra/tasks/cleanup.yml]: Ok

**Review by tombarron — COMMENTED** [review]
Would you add a paragraph to README.rst telling how to clean up?

Otherewise I think this is ready to merge once the path and resource renaming are fixed.  Sorry to be moving those around while you were doing this.

**Comment by vkmc** [comment]
Addressed comments

**Review by tombarron — APPROVED** [review]

**Pull request merged**